### ISSUE TITLE
v1.78a - 1.82 (Windows):  --algo ETHASH / ETCHASH / UBQHASH and --dualmode KASPADUAL not working with RX5700

### ISSUE THREAD

**Issue by UselessGuru** [Lolliedieb/lolMiner-releases#1987]
`lolMiner-v1.78a>lolminer.exe --algo ETCHASH --dualmode KASPADUAL --pool europe.etchash-hub.miningpoolhub.com:20615 --user *************************.************** --pass x --tls off --ethstratum ETHV1 --dualpool eu.mining.prohashing.com:3365 --dualuser *************************.************** --dualpass a=kheavyhash,n=Blackbox --maxdualimpact * --dualtls off --log off --apiport 4001 --shortstats 1 --longstats 5 --digits 6 --watchdog exit --dns-over-https 1 --devicesbypcie --devices 9:0`

Mining does not start mining until watchdog kicks in:
```

+---------------------------------------------------------+
|   _       _ __  __ _                   _   _____  ___   |
|  | | ___ | |  \/  (_)_ __   ___ _ __  / | |___  |( _ )  |
|  | |/ _ \| | |\/| | | '_ \ / _ \ '__| | |    / / / _ \  |
|  | | (_) | | |  | | | | | |  __/ |    | |_  / / | (_) | |
|  |_|\___/|_|_|  |_|_|_| |_|\___|_|    |_(_)/_/   \___/  |
|                                          Bugfix Version |
|                                                         |
|              This software is for mining                |
|              Autolykos V2                               |
|              Ethash, Etchash                            |
|              Equihash 144/5, 192/7, 210/9               |
|              BeamHash III                               |
|              NexaPow                                    |
|              Flux (ZelHash)                             |
|              Heavyhash-Karlsen                          |
|              Blake3 (Ironfish & Alephium)               |
|              Cuck(ar)oo 29                              |
|              Cuckaroo   30 CTX                          |
|              Cuckatoo   31/32                           |
|                                                         |
|                                                         |
|            Made by Lolliedieb, December 2023            |
+---------------------------------------------------------+

Setup Miner...
OpenCL driver detected.
Number of OpenCL supported GPUs: 2
Cuda driver detected.
Number of Cuda supported GPUs: 3
Device 0:
    Name:    Radeon RX 580 Series
    Address: 1:0
    Vendor:  Advanced Micro Devices (AMD)
    Drivers: OpenCL
    Memory:  8192 MByte (8128 MByte free)
    Active:  false (not selected)
Device 1:
    Name:    AMD Radeon RX 5700 XT
    Address: 9:0
    Vendor:  Advanced Micro Devices (AMD)
    Drivers: OpenCL
    Memory:  8176 MByte (8112 MByte free)
    Max epoch (approx): 886
    Active:  true (Selected Algorithm: ETHash OpenCL)
Device 2:
    Name:    NVIDIA GeForce GTX 1660 SUPER
    Address: 11:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  6143 MByte
    Active:  false (not selected)
Device 3:
    Name:    NVIDIA GeForce GT 1030
    Address: 12:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  2047 MByte
    Active:  false (not selected)
Device 4:
    Name:    NVIDIA GeForce GTX 750 Ti
    Address: 16:0
    Vendor:  NVIDIA Corporation
    Drivers: Cuda
    Memory:  2047 MByte
    Active:  false (not selected)

Applying overclock settings...

Connecting to pool...
DNS over HTTPS resolve succeeded
DNS over HTTPS resolve succeeded
Connected to eu.mining.prohashing.com(195.3.222.217):3365  (TLS disabled)
New extra nonce received: a221b1
Authorized worker: ************************
New target received: 000000003fffffff (Diff 4)
Connected to europe.etchash-hub.miningpoolhub.com(108.61.78.226):20615  (TLS disabled)
Set Ethash stratum mode: EthereumStratum/1.0.0
Subscribed to stratum server
New target received: 0000000055555553 (Diff 3.0000000027939677)
New job received: c1e5 Epoch: 628 (ETCHash epoch: 314) Difficulty: 12.88G
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
Start Mining...
Authorized worker: *****************************.******************
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
-------------------------------------------------------
  Generated light cache for epoch 628 (ETCHash: 314)
       Took 1794 ms, size of new DAG: 3535 MByte
-------------------------------------------------------
GPU 1: Starting DAG generation (normal mode)
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
-----------------------------------------------
Statistics (10:36:30); Uptime: 0h 0m 7s
lolMiner 1.78a, Nvidia 546.01, AMD CL 3516.0, Api port 4001
Algorithm 1: Etchash
Connected to: europe.etchash-hub.miningpoolhub.com:20615
Algorithm 2: HeavyHash-Kaspa
Connected to: eu.mining.prohashing.com:3365

      Name          Speed      Pool  Shares   Best    Eff.     Speed      Pool  Shares   Best    Eff.  Power  CCLK  MCLK  Core  Junc   Mem  Fan  Dual
                     Mh/s      Mh/s  A/S/Hw  Share  Mh/s/W      Mh/s      Mh/s     A/R  Share  Mh/s/W      W   MHz   MHz  Temp  Temp  Temp  Pct   Fct
GPU 1 RX 5700 XT 0.000000  0.000000   0/0/0    0.0   0.000  0.000000  0.000000     0/0    0.0   0.000   12.0     6   200    32    33    34   43  0.00
---------------------------
Total            0.000000  0.000000   0/0/0    0.0   0.000  0.000000  0.000000     0/0    0.0   0.000   12.0
-----------------------------------------------
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
GPU 1: DAG generation completed (4881 ms)
GPU 1: DAG verification passed
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
Etchash: Average speed (1s): 0.000000 Mh/s
HeavyHash-Kaspa: Average speed (1s): 0.000000 Mh/s
-----------------------------------------------
Statistics (10:36:35); Uptime: 0h 0m 12s
lolMiner 1.78a, Nvidia 546.01, AMD CL 3516.0, Api port 4001
Algorithm 1: Etchash
Connected to: europe.etchash-hub.miningpoolhub.com:20615
Algorithm 2: HeavyHash-Kaspa
Connected to: eu.mining.prohashing.com:3365

      Name          Speed      Pool  Shares   Best    Eff.     Speed      Pool  Shares   Best    Eff.  Power  CCLK  MCLK  Core  Junc   Mem  Fan  Dual
                     Mh/s      Mh/s  A/S/Hw  Share  Mh/s/W      Mh/s      Mh/s     A/R  Share  Mh/s/W      W   MHz   MHz  Temp  Temp  Temp  Pct   Fct
GPU 1 RX 5700 XT 0.000000  0.000000   0/0/0    0.0   0.000  0.000000  0.000000     0/0    0.0   0.000   39.6  2008   200    36    52    40   42  0.00
---------------------------
Total            0.000000  0.000000   0/0/0    0.0   0.000  0.000000  0.000000     0/0    0.0   0.000   39.6
--------------------------------------------

```

Same Issue with Etchash, Ethash or UbqHash

**Comment by UselessGuru** [comment]
Still broken in 1.79

**Comment by UselessGuru** [comment]
Resolved in 1.79

**Comment by UselessGuru** [comment]
Still not working in 1.81
